sureberus/constants.py

```diff
@@ -1 +1 @@
-_marker = object()
+class _marker(object): pass
```

Comment:
this is just a silly way to make the __repr__ of `_marker` show the name `_marker`, instead of it just showing as `<object at 0x...>`.